### DISCUSSION TITLE
Wazuh mail alerts multiple recipients

### DISCUSSION THREAD

**Discussion by DinhDiesel** [Security-Onion-Solutions/securityonion#8510] [Unsupported Versions]
I have my wazuh email alerts working, anything level 7 and above goes to one email with email log set to 1 and this works. Then I added a second email to receive email alerts regarding anything regarding specific groups and the second email does not receive any email regarding events within a specified group. I referred to the wazuh documentation and am unsure where I made a mistake. Below is my ossec.conf, any help would be much appreciated!

<ossec_config>
  8   <global>
  9     <jsonout_output>yes</jsonout_output>
 10     <alerts_log>yes</alerts_log>
 11     <logall>yes</logall>
 12     <logall_json>yes</logall_json>
 13     <email_notification>yes</email_notification>
 14     <smtp_server>x.x.x.x</smtp_server>
 15     <email_from>wazuh.securityOnion@domain.com</email_from>
 16     <email_to>working_email@domain.com</email_to>
 17     <email_maxperhour>60</email_maxperhour>
 18     <helo_server>securityOnion.domain.com</helo_server>
 19   </global>
 20
 21   <email_alerts>
 22     <email_to>thisEmail_doesNotWork@domain.com</email_to>
 23     < group >pci_dss_8.1.2,|pci_dss_10.2.5,</ group >
 24   </email_alerts>

**Comment by THExCAPNxRK** [comment]
I was able to get multiple emails to work by only using the `<email_alerts>` option and not the `<global>` settings.

So for you it would look something like this


```
<ossec_config>

<global>
<jsonout_output>yes</jsonout_output>
<alerts_log>yes</alerts_log>
yes
<logall_json>yes</logall_json>
<email_notification>yes</email_notification>
<smtp_server>x.x.x.x</smtp_server>
<email_from>wazuh.securityOnion@domain.com</email_from>
<email_maxperhour>60</email_maxperhour>
<helo_server>securityOnion.domain.com</helo_server>
</global>

<email_alerts>
<email_to>working_email@domain.com</email_to>
<level>7</level>
</email_alerts>

<email_alerts>
<email_to>thisEmail_doesNotWork@domain.com</email_to>
<group>pci_dss_8.1.2,|pci_dss_10.2.5,</group>
</email_alerts>
```

Hope this helps.
